src/game_api/script/lua_backend.cpp

```diff
@@ -176,10 +176,13 @@ bool LuaBackend::update()
         std::vector<Player*> players = get_players();
         lua["players"] = std::vector<Player*>(players);
 
-        if (g_state->screen != state.screen && g_state->screen_last != 5)
+        if (g_state->loading == 1 && g_state->loading != state.loading)
         {
             level_timers.clear();
             script_input.clear();
+        }
+        if (g_state->screen != state.screen && g_state->screen_last != 5)
```

Comment:
Instead here, why do we only check moving from options and not moving to options? Or check neither. As it's only a callback users might be interested in moving to the options screen.

Reply:
My brain really went backwards here didn't it if I put the options check in the only place it wasn't needed.